src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedNewTests.cs

```diff
@@ -1124,5 +1332,2216 @@ public static void Main()
+                // (10,17): error CS8310: Operator '^' cannot be applied to operand 'new(...)'
+                //         var e = 1 ^ new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 ^ new()").WithArguments("^", "new(...)").WithLocation(10, 17),
+                // (11,17): error CS8310: Operator '*' cannot be applied to operand 'new(...)'
+                //         var f = 1 * new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 * new()").WithArguments("*", "new(...)").WithLocation(11, 17),
+                // (12,17): error CS8310: Operator '/' cannot be applied to operand 'new(...)'
+                //         var g = 1 / new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 / new()").WithArguments("/", "new(...)").WithLocation(12, 17),
+                // (13,17): error CS8310: Operator '%' cannot be applied to operand 'new(...)'
+                //         var h = 1 % new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 % new()").WithArguments("%", "new(...)").WithLocation(13, 17),
+                // (14,17): error CS8310: Operator '>>' cannot be applied to operand 'new(...)'
+                //         var i = 1 >> new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 >> new()").WithArguments(">>", "new(...)").WithLocation(14, 17),
+                // (15,17): error CS8310: Operator '<<' cannot be applied to operand 'new(...)'
+                //         var j = 1 << new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 << new()").WithArguments("<<", "new(...)").WithLocation(15, 17),
+                // (16,17): error CS8310: Operator '>' cannot be applied to operand 'new(...)'
+                //         var k = 1 > new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 > new()").WithArguments(">", "new(...)").WithLocation(16, 17),
+                // (17,17): error CS8310: Operator '<' cannot be applied to operand 'new(...)'
+                //         var l = 1 < new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 < new()").WithArguments("<", "new(...)").WithLocation(17, 17),
+                // (18,17): error CS8310: Operator '>=' cannot be applied to operand 'new(...)'
+                //         var m = 1 >= new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 >= new()").WithArguments(">=", "new(...)").WithLocation(18, 17),
+                // (19,17): error CS8310: Operator '<=' cannot be applied to operand 'new(...)'
+                //         var n = 1 <= new();
+                Diagnostic(ErrorCode.ERR_BadOpOnTypelessExpression, "1 <= new()").WithArguments("<=", "new(...)").WithLocation(19, 17),
+                // (20,22): error CS9367: The default constructor of the value type 'int' may not be used with 'new(...)'; Use 'default' or a literal expr
```

Comment:
This is currently skipped. There's a problem with reporting errors for `new()` on conversion creation.

https://github.com/dotnet/roslyn/blob/b4f7a28aa65a9904121c3dcf02130bdf92f926d5/src/Compilers/CSharp/Portable/Binder/Binder_TupleOperators.cs#L27-L30

The diagnostics on `new()`'s target-type is captured and discarded here.